File: vuebugtrackerproject.client/src/enumConsts/ProjectType.ts
// Descibes the types of projects being search for.
const ProjectType = {
  //Gets all projects
  All: 0,
  //Only gets projects with no bugs that are open
  NoOpenBugs: 1,
  //Only gets projects with at least one open bug
  OpenBugsOnly: 2
} as const;

export default ProjectType;

File: VueBugTrackerProject.Classes/Enums/ProjectType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace VueBugTrackerProject.Classes
{
    /// <summary>
    /// Descibes the types of projects being search for.
    /// </summary>
    public enum ProjectType
    {
        /// <summary>
        /// Gets all projects.
        /// </summary>
        All,
        /// <summary>
        /// Only gets projects with no bugs that are open.
        /// </summary>
        NoOpenBugs,
        /// <summary>
        /// Only gets projects with at least one open bug.
        /// </summary>
        OpenBugsOnly
    }
}
